File: Server/WebServer/Models/Movie.cs
namespace WebServer.Models
{
    public class Movie
    {
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Director { get; set; } = string.Empty;
        public int Year { get; set; }
        public string Description { get; set; } = string.Empty;
        public string ImageUrl { get; set; } = string.Empty;
        public string Url { get; set; } = string.Empty;
    }
}

File: Labos4/app/src/main/java/hr/tvz/android/listalapenda/models/Movie.kt
package hr.tvz.android.listalapenda.models

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.parcelize.Parcelize

@Parcelize
@Entity(tableName = "movies")
data class Movie(
    @PrimaryKey val id: Int,
    val title: String,
    val director: String,
    val year: Int,
    val description: String,
    val imageUrl: String,
    val url: String
) : Parcelable